src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/WindowsRuntime/Windows/UI/ViewManagement/NativeMethods.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//
+//
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Security;
+using MS.Internal.PresentationCore;
+
+namespace MS.Internal.WindowsRuntime
+{
+    namespace Windows.UI.ViewManagement
+    {
+        /// <summary>
+        /// Contains internal RCWs for invoking the InputPane (tiptsf touch keyboard)
+        /// </summary>
+        internal static class NativeMethods
+        {
+            [DllImport(DllImport.ApiSetWinRTString, CallingConvention = CallingConvention.StdCall)]
+            internal static extern unsafe int WindowsCreateString([MarshalAs(UnmanagedType.LPWStr)] string sourceString,
+                                                  int length,
+                                                  out IntPtr hstring);
+
+            [DllImport(DllImport.ApiSetWinRTString, CallingConvention = CallingConvention.StdCall)]
+            internal static extern int WindowsDeleteString(IntPtr hstring);
+
+            [DllImport(DllImport.ApiSetWinRT)]
```

Comment:
Why no CallingConvention here?

Reply:
No particular reason. Looks like I missed it when I copied it over. The default is correct, but I'll add it explicitly.